packages/devtools_app/lib/src/screens/dtd/services.dart

```diff
@@ -0,0 +1,305 @@
+        ...response.dtdServices.map((value) {
+          // If the DTD service has the form 'service.method', split up the two
+          // values. Otherwise, leave the service null and use the entire name
+          // as the method.
+          String? service;
+          String method;
+          final parts = value.split('.');
+          if (parts.length > 1) {
+            service = parts[0];
+          }
+          method = parts.last;
+          return DtdServiceMethod(service: service, method: method);
+        }),
+        for (final service in response.clientServices) ...[
+          for (final method in service.methods.values)
+            DtdServiceMethod(
+              service: service.name,
+              method: method.name,
+              capabilities: method.capabilities,
+            ),
+        ],
+      ];
+    });
+  }
+}
+
+class ServicesView extends StatelessWidget {
+  const ServicesView({super.key, required this.controller});
+
+  final ServicesController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        return FlexSplitColumn(
+          totalHeight: constraints.maxHeight,
+          initialFractions: const [0.6, 0.4],
+          minSizes: const [100.0, 200.0],
+          headers: [
+            AreaPaneHeader(
+              title: Text('Registered services', style: theme.boldTextStyle),
+              roundedTopBorder: false,
+              includeTopBorder: false,
+              tall: true,
+              actions: [
+                DevToolsButton.iconOnly(
+                  icon: Icons.refresh,
+                  onPressed: controller.refresh,
+                ),
+              ],
+            ),
+            AreaPaneHeader(
+              title: Text('Manually call service', style: theme.boldTextStyle),
+              roundedTopBorder: false,
+              tall: true,
+            ),
+          ],
+          children: [
+            ValueListenableBuilder(
+              valueListenable: controller._services,
+              builder: (context, services, child) {
+                return ValueListenableBuilder<DtdServiceMethod?>(
+                  valueListenable: controller._selectedService,
+                  builder: (context, selectedService, _) {
+                    final sortedServices = services.toList()..sort();
+                    return Scrollbar(
+                      thumbVisibility: true,
+                      child: ListView.builder(
+                        primary: true,
+                        itemCount: sortedServices.length,
+                        itemBuilder: (context, index) {
+                          final service = sortedServices[index];
+                          return ListTile(
+                            title: Text(
+                              service.displayName,
+                              style: Theme.of(context).regularTextStyle,
```

Comment:
There's already a `theme` variable at the top of the build method that should probably be used here.